_unittest/test_26_emit.py

```diff
@@ -1,11 +1,17 @@
 # Setup paths for module imports
 import gc
+from pyaedt.modeler.PrimitivesEmit import EmitComponent, EmitComponents
```

Comment:
@myoung301 This import could be located under "# Import required modules" with the other pyaedt import.